index.html

```diff
@@ -13,6 +13,50 @@
   </head>
 
   <body>
-   
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name">Kevin Toro</p>
```

Comment:
This must be `a` tag. This would be a clickable link, That always take us back to the `homepage`.
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.